modules/oe_content_entity/modules/oe_content_entity_contact/oe_content_entity_contact.post_update.php

```diff
@@ -108,7 +108,6 @@ function oe_content_entity_contact_post_update_00006(): void {
  * Create the Contact view mode and reference field storage.
  */
 function oe_content_entity_contact_post_update_00007(): void {
-  \Drupal::service('module_installer')->install(['entity_reference_revisions']);
```

Comment:
This module is still listed as a dependency and it's in the README, we should require the entity_reference and enable it too in this update hook, I think.

![image](https://user-images.githubusercontent.com/153362/100852543-f4aa5e00-3486-11eb-9f80-91136e94c751.png)